packages/config/config/devices/0x0234/zdb5100.json

```diff
@@ -4,824 +4,494 @@
 	"manufacturer": "Logic Group",
 	"manufacturerId": "0x0234",
 	"label": "ZDB5100",
-	"description": "MATRIX Switch with Dimmer and Backlight",
+	"description": "Wall Switch",
 	"devices": [
 		{
 			"productType": "0x0003",
-			"productId": "0x0121"
+			"productId": "0x0121",
+			"zwaveAllianceId": 3399
 		}
 	],
 	"firmwareVersion": {
 		"min": "0.0",
 		"max": "255.255"
 	},
-	"supportsZWavePlus": true,
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"description": "Sends Device Reset notifications, and Central Scene notifications.",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic Report, Pushbutton 1",
```

Comment:
Please make the association labels consistent between the files

Reply:
resolved